client/src/components/TreeDropdownField/TreeDropdownField.js

```diff
@@ -405,10 +423,15 @@ class TreeDropdownField extends Component {
    */
   renderMenu(renderMenuOptions) {
     // Build root node
-    const tree = this.getVisibleTree() || {};
+    const visibleTree = this.getVisibleTree();
```

Comment:
Oh, I was wondering if you wanted the visibleTree to be one level deeper (i.e. another dummy level for the root object) but I realised I was mis-understanding the point of the api change. :)